components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/impl/ApisApiServiceImpl.java

```diff
@@ -501,6 +503,44 @@ public Response apisApiIdDocumentsGet(String apiId, Integer limit, Integer offse
         return Response.ok().entity(documentListDTO).build();
     }
 
+    @Override
+    public Response apisApiIdGenerateSdkLanguagePost(String apiId, String language, Request request)
+            throws NotFoundException {
+        if (StringUtils.isBlank(apiId) || StringUtils.isBlank(language)) {
+            return Response.status(Response.Status.BAD_REQUEST)
+                    .entity("API ID or language is not valid")
```

Comment:
Please use a DTO when sending error response. Just sending plain text is not good. Refer the how error response is generated in } catch (APIManagementException e) { below. Also use application/json content type.